src/api/data/charts.go

```diff
@@ -8,32 +8,47 @@ import (
 	"github.com/helm/monocular/src/api/pkg/swagger/models"
 )
 
-// GetChart gets the chart associated with the passed-in repo+name
-func GetChart(repo, name string) (models.Resource, error) {
-	// TODO implement actual "get chart" business logic
+// Charts is an interface for managing chart data
+type Charts interface {
```

Comment:
Why are all of these `Get*()`? Couldn't they just be `Chart` (or `Get`), `All`, `AllFromRepo`